caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -114,7 +121,7 @@ def Train(self, request, *_, **__) -> TrainingJob:
 
         try:
             with alog.ContextLog(log.debug, outer_scope_name):
-                # BlocksSampleTaskSampleModuleTrainRequest
+                # ModulesSampleTaskSampleModuleTrainRequest
```

Comment:
This looks like dead code?

Reply:
I think this is an example of how we get to the module name itself?